### PR TITLE
Add implicit #bulk_management including filter

### PR DESCRIPTION
**Pull request by sauloperez**
#### What? Why?

OFN it's hard enough. No need to abuse implicitness making things very
hard to follow.

I've spent around 20min trying to find out where this controller action
was implemented until I realized Rails renders the matching view if no
controller action is defined.

Making it git-greppable makes it a bit easier next time.

#### What should we test?

`/admin/orders/bulk_management` should render the appropriate information when using the filters.

#### Release notes

Added implicit `bulk_management` controller action to increase readability.

Changelog Category: Changed

### CODE REVIEW

**Comment by sauloperez** [comment]
yes... and it's not even the first time I have one of those moments. I think this particularity is very confusing...

**Comment by JenRSmith** [comment]
Testing completed and all working as it should be.

Testing notes here - https://docs.google.com/document/d/1wrjUfOFd6qVSLB2oRQbM3Ouh-q-5hOyCp7CtiDwnxAg/edit#heading=h.lpnhrpqaw84x

**Comment by sigmundpetersen** [comment]
@JenRSmith where did you deploy this for testing? From your testing notes it says Katuma staging but I can't see that it was staged there.

**Comment by JenRSmith** [comment]
@sigmundpetersen Yes, Katuma staging. This was the first one I did without Dani present so perhaps I missed something. Let me check with her that I followed the correct procedure.

**Comment by luisramos0** [comment]
no, it wasnt staged on katuma in the last few days, you can see the deploy history here: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/servers/katuma-staging/deploys/135

**Comment by sauloperez** [comment]
It can be tested again by staging `master` @JenRSmith . Would you mind?

**Comment by daniellemoorhead** [comment]
@sauloperez neither myself nor @JenRSmith know how to stage master. You'll need to explain how to do that, or, stage it for us and then Jen can retest...

**Comment by sigmundpetersen** [comment]
Go to the master branch https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master, click on the latest build and then 'Deploy manually'

**Comment by sauloperez** [comment]
I feel this deserves a "Staging branches" wiki page

**Comment by JenRSmith** [comment]
@sauloperez I have staged to master as per instructions by @sigmundpeterson. Deployed to Katuma & all displaying correctly. Thanks for the assistance with this one.

**Comment by sauloperez** [comment]
Awesome. Thank you!

El dv., 20 de set. 2019, 5:08, JenRSmith <notifications@github.com> va
escriure:
